packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -53,6 +55,17 @@ const Duration _kCursorBlinkWaitForStart = Duration(milliseconds: 150);
 // is shown in an obscured text field.
 const int _kObscureShowLatestCharCursorTicks = 3;
 
+/// A map used to disable scrolling shortcuts in text fields.
+///
+/// This is a temporary fix for: https://github.com/flutter/flutter/issues/79012
```

Comment:
If this is temporary, is there a way we can do this such that it doesn't leak into the public API? Also, let's file an issue to implement a permanent solution.

Reply:
BTW, I'd use `//` instead of `///` for the "This is a temporary fix" line.

Reply:
> is there a way we can do this such that it doesn't leak into the public API?

We can make the map private and duplicate it in two files. Since this is a temporary workaround, I think it's fine to duplicate for now.

Reply:
Yeah, this is good point: we shouldn't make this public.